CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+**Table of Contents**
+- [Getting Started](#getting-started) (1st draft)
+- [Philosophy](#philosophy) (TBD)
+- [Reporting an Issue](#reporting-an-issue) (TBD)
+- [Pull Requests](#pull-requests) (1st draft)
+	-  _Should this be renamed to "fixing an issue"?_
```

Comment:
No objection... `Fixing an Issue (via Pull Request)`?

Reply:
That sounds too verbose to me.

Reply:
`Fixing an Issue (PR)`?